packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx

```diff
@@ -60,32 +59,24 @@ const EditCurrentEvent: React.FC<EditCurrentEventParams> = ({
   ];
 
   const validateArbiterInfo = (arbiterInfo: ArbiterInfo): boolean => {
-    if (arbiterInfo.arbiterToken == '') {
-      showError('Arbiter Email token cannot be blank');
-      return false;
-    }
-    if (arbiterInfo.divisionId == '') {
-      showError('Division Id cannot be blank');
-      return false;
-    }
-    if (arbiterInfo.eventName == '') {
-      showError('Event Name cannot be blank');
-      return false;
-    }
-    if (arbiterInfo.userName == '') {
-      showError('Arbiter Name cannot be blank');
-      return false;
-    }
-    if (arbiterInfo.userId == '') {
-      showError('Arbiter Id cannot be blank');
-      return false;
-    }
-    if (arbiterInfo.pin == '') {
-      showError('Arbiter Pin cannot be blank');
-      return false;
-    }
-    if (!validUrl(arbiterInfo.broadcastUrl)) {
-      showError('Invalid Pgn Url');
+    const validationError = Object.entries(arbiterInfo)
+      .map(([fieldName, fieldValue]) => {
+        if (fieldName === 'broadcastUrl') {
+          if (!validUrl(fieldValue)) {
+            return `Error, broadcast url is not in correct format`;
```

Comment:
Continuing my trend of asking for really small grammatical changes, could this please be 'Error, the broadcast URL is not in the correct format. '?